submissions/unabyband/oop_exercise/index.js

```diff
@@ -0,0 +1,109 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/unabyband/a-tiny-JS-world
+   Web app: https://unabyband.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+  constructor(species, name, gender, legs, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.saying = saying;
+    this.friends = friends;
+  }
+
+  printInhabitant() {
+    //I've decided to form friends parameter separately from others just in order to improve code reading
+    //But it also could be made directly in arguments of function like in the overridden method below
+    const friendsToPrint = this.friends
+      ? this.friends.map((friend) => friend.name).join(", ")
+      : `${this.species} has no friends!`;
+  
+    this.printToPage(this.species, 
+      `<strong>${this.name}</strong>`, 
+      this.gender, 
+      `On ${this.legs} legs`, 
+      `<em>${this.saying}</em>`, 
+      friendsToPrint);
+  }  
+  
+  printToPage() {
+    print([...arguments].join("; "));
+  }
+}
+
+class Human extends Inhabitant {
+  constructor(name, gender, saying, friends) {
+    const legs = 2;
+    const hands = 2;
+    super("human", name, gender, legs, saying, friends);
+    this.hands = hands;
+  }
+
+  printInhabitant() {
+    super.printToPage(this.species,
```

Comment:
Why not re-use parent's printInhabitant and then adding what is specific to this class?
I see, because parent's `printInhabitant` is not really re-usable as it returns nothing.
Use `print` function completely outside of classes definition.

Reply:
I've made it, please review again